scripts/bib2md.py

```diff
@@ -0,0 +1,163 @@
+# Install the library before, with the following command:
+# pip install bibtexparser --pre
+
+# to run on the CLI:
+# python bib2md.py -f reference_file.bib -ord desc|asc
+
+import bibtexparser
+import sys
+import argparse
+
+def format_simple(entry_str, order):  
+    library = bibtexparser.parse_string(entry_str)
+    formatted_entries = []
+    unprocessed_entries = []
+    
+    for entry in library.entries:
+        try:

```

Comment:
All this loop body should be a separate function that takes a single entry as input and returns a string as output.
To signal that the entry could not be processed you can return None, or empty string, or better, throw a custom exception (something like UnsupportedEntryTypeException)